packages/api/endpoints/execution-status.js

```diff
@@ -91,6 +101,23 @@ async function get(req, res) {
     }
   }
 
+  // include associated granules
+  const granuleCumulusIds = await getApiGranuleExecutionCumulusIdsByExecution(knex, [response]);
+  const granules = await granulePgModel.searchByCumulusIds(knex, granuleCumulusIds);
+  const apiGranules = await Promise.all(granules
+    .map(async (pgGranule) => {
+      const pgCollection = await collectionPgModel.get(
+        knex,
+        { cumulus_id: pgGranule.collection_cumulus_id }
+      );
+
+      return await translatePostgresGranuleToApiGranule({
+        granulePgRecord: pgGranule,
+        collectionPgRecord: pgCollection,
+        knexOrTransaction: knex,
+      });
+    }));
+
```

Comment:
This is the change required in order to provide associated granules so a link to associated executions can be provided on the dashboard.